packages/components/addon/components/hds/pagination/nav/index.hbs

```diff
@@ -1,10 +1,16 @@
+{{! template-lint-disable simple-unless }}
 <nav class="hds-pagination-nav" aria-label="Pagination navigation" ...attributes>
-  <Hds::Pagination::Nav::ButtonArrow
-    @direction="prev"
-    @hideLabel={{eq this.type "numbered"}}
-    @onClick={{this.onPageChange}}
-    @disabled={{this.isDisabledPrev}}
-  />
+  {{! TODO use `has-block-params`? https://api.emberjs.com/ember/4.4/classes/Ember.Templates.helpers/methods/action?anchor=has-block-params }}
+  {{#unless (has-block)}}
```

Comment:
I think `has-block` is what we want here, as this would cover the cases where we have contextual components (`<N.ButtonPrev` and `N.ButtonNext`).

`<Hds::Pagination::Nav as |N|>` would return true for `has-block-params` but so will`<Hds::Pagination::Nav @totalPages={{10}} @currentPage={{3}} />`, so it's not probably what we want.

Reply:
Outdated.